### ISSUE TITLE
httpsplitting that proxy_pass leads to can't be used

### ISSUE THREAD

**Issue by JoyChou93** [yandex/gixy#75]
such the config

```

	server {
		server_name act.baidu.com;

		location ~* ^/test/(.*) {
			proxy_pass http://10.10.10.10/$1;
		}
	}
```

**Comment by buglloc** [comment]
Some upstream servers perceive CR as a delimiter of request headers (in the likeness of browsers), but you are right that there are very few of them. Therefore, this checks are temporary disabled for proxied requests, I will later return with the level of the INFO level.

**Comment by buglloc** [comment]
Fixed in Gixy v0.1.9
